Flask 2.2.0 Incompatibility

I'm running a github ci check on [my project](https://github.com/z-t-y/Djask) when I encounter this issue:

```
tests/conftest.py:3: in <module>
    from djask import Djask
src/djask/__init__.py:6: in <module>
    from .app import Djask as Djask
src/djask/app.py:5: in <module>
    from apiflask import APIFlask
/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/apiflask/__init__.py:3: in <module>
    from .app import APIFlask as APIFlask
/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/apiflask/app.py:35: in <module>
    from .route import route_patch
/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/apiflask/route.py:3: in <module>
    from flask.views import MethodViewType
E   ImportError: cannot import name 'MethodViewType' from 'flask.views' (/opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/flask/views.py)
```

Environment:

- Python version: 3.10.5
- Flask version: 2.2.0
- APIFlask version: 1.1.0

--- a/src/apiflask/route.py
+++ b/src/apiflask/route.py
@@ -1,6 +1,6 @@
 import typing as t
 
-from flask.views import MethodViewType
+from flask.views import MethodView
 
 from .openapi import get_path_description
 from .openapi import get_path_summary
@@ -79,7 +79,7 @@ def route_patch(cls):
             # a function returned by MethodViewClass.as_view()
             is_view_class = True
             view_class = view_func.view_class  # type: ignore
-        elif isinstance(view_func, MethodViewType):
+        elif isinstance(view_func, type(MethodView)):
             # a MethodView class passed with the route decorator
             is_view_class = True
             view_class = view_func  # type: ignore

--- a/src/apiflask/scaffold.py
+++ b/src/apiflask/scaffold.py
@@ -7,7 +7,7 @@ import flask
 from flask import current_app
 from flask import jsonify
 from flask import Response
-from flask.views import MethodViewType
+from flask.views import MethodView
 from marshmallow import ValidationError as MarshmallowValidationError
 from webargs.flaskparser import FlaskParser as BaseFlaskParser
 from webargs.multidictproxy import MultiDictProxy
@@ -116,7 +116,7 @@ class APIScaffold:
             raise RuntimeError('Use the "route" decorator to use the "methods" argument.')
 
         def decorator(f):
-            if isinstance(f, MethodViewType):
+            if isinstance(f, type(MethodView)):
                 raise RuntimeError(
                     'The route shortcuts cannot be used with "MethodView" classes, '
                     'use the "route" decorator instead.'

